cli/src/pcluster/models/cluster.py

```diff
@@ -324,7 +344,7 @@ def create(
                 )
 
             # upload cluster artifacts and generated template
-            self._upload_artifacts()
+            artifacts_uploaded = self._upload_artifacts()
```

Comment:
rather than having this return a bool we could just set artifacts_uploaded to True right after calling the function?

Reply:
Changed